inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -506,7 +506,7 @@ struct jit_uni_mvn_kernel_f32 : public jit_uni_mvn_kernel, public jit_generator
     Xbyak::Reg64 reg_d_bias = rdx;
 
     Xbyak::Reg64 reg_load_table = r15;
-    Xbyak::Reg64 reg_load_store_mask = rcx;
+    Xbyak::Reg64 reg_load_store_mask = rbp;
```

Comment:
Just wondering how the impl on master is working on Windows? Do we cover this path by tests?

Reply:
Thank you for this question and I re-checked the code.
reg_load_store_mask is only used on avx512 target for [mask load/store](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/emitters/jit_load_store_emitters.cpp#L158) for tails.
Other target use [progressive load/store](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/emitters/jit_load_store_emitters.cpp#L200) for tails.
So we just care about avx512. On this target rcx used and only used as abi_param1 at the very beginning of execution loop to load parameters. Then we can reuse rcx in the execution loop within emit_code() to load data.
Tests already cover this case such as input tensor of [dimension [2, 19, 5, 10]](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/single_layer_tests/mvn.cpp#L88), 5*10 is not devisable by 16, which is processed in one kernel in planar layout. The mask loaded for tails on avx512 happens here.
So `reg_load_store_mask = rcx` does not lead to bug, changing just for remove possible worries and readability and meanwhile rbp is available in this node.

Reply:
I see. Thanks for explanation.